app/models/user.rb

```diff
@@ -3,4 +3,13 @@ class User < ApplicationRecord
   has_many :user_games, dependent: :destroy
   has_many :games, through: :user_games
   validates_presence_of :name, :email, :google_token, :photo
+
+  def self.from_omniauth(auth)
+    find_or_create_by(email: auth.info.email) do |user|
```

Comment:
`find_or_create_by` finds the first email that matches the user logging in, or creates a user if it doesn't already exist in our db

Reply:
That's great to know.

Reply:
nice. feels a bit like `||=`